tensorflow-xla-runtime.spec

```diff
@@ -0,0 +1,32 @@
+### RPM external tensorflow-xla-runtime 2.12.0
+
+Requires: eigen tensorflow-sources
+BuildRequires: cmake
+
+%prep
+unzip -q -o ${TENSORFLOW_SOURCES_ROOT}/tensorflow-%{realversion}-cp%{cms_python3_major_minor}-cp%{cms_python3_major_minor}-linux_%{_arch}.whl
+
+%build
+
+# fix a single comparison between size_t and int
+sed -i -r 's/assert\(\(arg_size\(index\)/assert\(\(\(size_t\)arg_size\(index\)/' tensorflow/include/tensorflow/compiler/tf2xla/xla_compiled_cpu_function.h
+
+export CPATH="${CPATH}:${EIGEN_ROOT}/include/eigen3"
+export CPATH="${CPATH}:%{i}/tensorflow/include"
+export CPATH="${CPATH}:%{i}/tensorflow/include/third_party/eigen3"
+
+pushd tensorflow/xla_aot_runtime_src
+  cmake . -DCMAKE_CXX_FLAGS="-fPIC"
```

Comment:
In order to properly build for multi-vectorization please add `-msse3` here e.g. `-DCMAKE_CXX_FLAGS="-fPIC -msse3"`